src/main/java/catan/services/util/play/PlayUtil.java

```diff
@@ -52,6 +54,16 @@ public void updateAvailableUserActions(GameBean game) throws GameException {
         }
     }
 
+    public void finishGameIfTargetReached(GameBean game) {
+        for (GameUserBean gameUser : game.getGameUsers()) {
```

Comment:
why do we iterate all player? only one plaer can reach victory points, and it is the player, that made action